docs/user-guide/ze-cics.md

```diff
@@ -101,6 +133,20 @@ To show more than one CICS profiles in the tree, select the **+** button and cho
 
 ### Deleting profiles
 
+#### Using Zowe Team Configuration
+
+1. Right-click a chosen profile.
+
+2. Select **Delete Profile** which opens the config file.
+
+3. Edit the config file to remove the cics profile entry.
```

Comment:
Similar to line #76: It might be implicit, but i have been told that the user needs to save the file before the refresh. Suggest adding a step.

Save the file.